python/ray/serve/tests/test_autoscaling_policy.py

```diff
@@ -178,7 +178,7 @@ def test_e2e_basic_scale_up_down(serve_instance):
     @serve.deployment(
         _autoscaling_config={
             "metrics_interval_s": 0.1,
-            "min_replicas": 1,
+            "min_replicas": 0,
```

Comment:
Can we parametrize this test with `min_replicas=0` and `min_replicas=1`? It would help us distinguish whether future changes break only scale-to-zero or autoscaling in general.

Reply:
good idea